MIP39/MIP39c2-Subproposals/MIP39c2-SP7.md

```diff
@@ -6,6 +6,7 @@
 MIP39c2-SP#: 7
 Author(s): Protocol Engineering Core Unit
 Contributors: Smart Contracts Domain Team
+Tags: core-unit, mip-set, smart-contracts, protocol-engineering-mipset
```

Comment:
protocol-engineering-mipset doesn't follow the same format as the previous tags like 

"MDS-001-mipset"